dev/gaptest.expect

```diff
@@ -1,4 +1,4 @@
-set timeout 10
+set timeout 60
```

Comment:
Is this still necessary?

Reply:
Nope, I took it out. If we ever notice this failing we could boost the limit again, but hopefully 10 seconds will be enough.